bears/python/PyromaBear.py

```diff
@@ -0,0 +1,40 @@
+import os.path
+from shutil import which
+from pyroma import projectdata
+from pyroma.ratings import rate
+
+from coalib.bears.GlobalBear import GlobalBear
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+
+
+class PyromaBear(GlobalBear):
+    LANGUAGES = {'Python', 'Python 3'}
+    REQUIREMENTS = {PipRequirement('pyroma', '2.2.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    @classmethod
+    def check_prerequisites(cls):
+        return ('Pyroma is missing. Make sure to install it using '
+                '`pip3 install pyroma`.'
+                if which('pyroma') is None else True)
+
+    def run(self):
+        """
+        Checks for Python packaging best practices using `pyroma`.
+
+        See <https://bitbucket.org/regebro/pyroma/> for more information.
+        """
+
+        setup_file = os.path.join(self.get_config_dir(), 'setup.py')
```

Comment:
I'd rather take all `setup.py` files that is in the file_dict, we don't want any file to be checked if it's not in the file_dict!